Update resource-manager-common-deployment-errors.md

przlplx opened the pull request:
Per Zainab Hakim: Added a third link in the "AllocationFailed" row under "More information": "Troubleshoot allocation failures when you create, restart, or resize Windows VMs in Azure"

azurecla commented:
Hi __@dtriana__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@dtriana : Thanks for your contribution to the Azure documentation! The author, @viv-liu, has been notified to review your proposed change.

viv-liu commented:
#sign-off

przlplx committed:
Update resource-manager-common-deployment-errors.md

Per Zainab Hakim: Added a third link in the "AllocationFailed" row under "More information": "Troubleshoot allocation failures when you create, restart, or resize Windows VMs in Azure"

v-shils committed:
Merge branch 'master' into patch-1
